bin/tag-release.sh

```diff
@@ -19,9 +19,25 @@ while [[ $# -ge 1 ]]; do
   shift # past argument or value
 done
 
+# compare master and stage
+echo "Comparing master branch to staging..."
+git fetch "$moz_git_remote"
+stage_hash=`git rev-parse $moz_git_remote/stage`
```

Comment:
It's more reliable and portable to use `$()` instead of the back-ticks. 

```suggestion
stage_hash=$(git rev-parse "${moz_git_remote}/stage")
```